packages/state-transition/src/block/processAttestationsAltairPlus.ts

```diff
@@ -28,7 +29,8 @@ const TIMELY_TARGET = 1 << TIMELY_TARGET_FLAG_INDEX;
 const TIMELY_HEAD = 1 << TIMELY_HEAD_FLAG_INDEX;
 const SLOTS_PER_EPOCH_SQRT = intSqrt(SLOTS_PER_EPOCH);
 
-export function processAttestationsAltair(
+export function processAttestationsAltairPlus(
```

Comment:
Should this be called by fork name?
```suggestion
export function processAttestationsDeneb(
```

Reply:
umm.. it also processes post altair pre deneb attestations (so a unified fn for processing attestations on/after altair) like how we handle everything else in a generic way (except phase0 which are so different , so we can let it stay separate)

Reply:
how about `processAttestationsMultiFork`?